ebos/eclwriter.hh

```diff
@@ -121,6 +123,9 @@ class EclWriter : public EclGenericWriter<GetPropType<TypeTag, Properties::Grid>
     enum { enableSolvent = getPropValue<TypeTag, Properties::EnableSolvent>() };
 
 public:
+
+    Inplace inplace_;
```

Comment:
The object **probably** does not need to be `public:`, so maybe we can put it alongside the writer's other [data members](https://github.com/OPM/opm-simulators/blob/eec2fbaaa13a65f33fbf19a4c563d8be63ea50ce/ebos/eclwriter.hh#L670-L673) instead?